### PR TITLE
Implement November Seasonal, Super Squad Showdown

### PR DESCRIPTION
**Pull request by ascriptmaster**
I'm dead

I'll create a few more custom sets later (maybe after the work week is over)

### CODE REVIEW

**Comment by TheImmortal** [comment]
Change the seasonal alias too

**Comment by ascriptmaster** [comment]
The what?

Also I forgot the ruleset. Silly me!

**Comment by ascriptmaster** [comment]
ok I saw it

**Comment by ascriptmaster** [comment]
Scarecrow now has a custom set with Gunk Shot in it.

File: data/scripts.js [hunk 1]
```diff
@@ -3150,6 +3150,91 @@ exports.BattleScripts = {
...
+			['Magneto', 'Mystique', 'Quicksilver', 'Scarlet Witch', 'Blob', 'Pyro', 'Sabretooth', 'Juggernaut', 'Toad'],
+			['Captain America', 'Hulk', 'Iron Man', 'Hawkeye', 'Black Widow', 'Thor', 'Nick Fury', 'Vision'],
+			['Starlord', 'Gamora', 'Groot', 'Rocket Raccoon', 'Drax the Destroyer', 'Nova'],
+			['Batman', 'Superman', 'Aquaman', 'Wonder Woman', 'Green Lantern', 'The Flash', 'Green Arrow', 'Firestorm'],
+			['Robin', 'Starfire', 'Cyborg', 'Beast Boy', 'Raven', 'Jinx', 'Terra', 'Blue Beetle'],
+			['Mr. Fantastic', 'Invisible Woman', 'Thing', 'Human Torch', 'Spiderman', 'Ant-Man'],
+			['Baymax', 'Honey Lemon', 'GoGo Tomago', 'Wasabi-no-Ginger', 'Fredzilla', 'Silver Samurai', 'Sunfire'],
+			['Joker', 'Deadshot', 'Harley Quinn', 'Boomerang', 'Killer Croc', 'Enchantress'],
+			['Poison Ivy', 'Bane', 'Scarecrow', 'Two-Face', 'Penguin', 'Mr. Freeze', 'Catwoman']
+			//,['Lex Luthor', 'Darkseid', 'General Zod', 'Braniac', 'Doomsday', 'Bizarro', 'Atomic Skull']
```
Comment (Joimer) [inline data/scripts.js]: This line should be deleted if it's not getting in in the case of a merge.

**Comment by Joimer** [comment]
Moving this seasonal to november

**Comment by ascriptmaster** [comment]
Last changes before vacation.

Some notes because I think balancing might be needed in the future:

- Wonder Woman is a tad strong usually. Scrappy HJK 2fast2furious
- Big Hero 6 has 4 Pokemon weak to Fighting, which makes them very weak to the aforementioned Wonder Woman. Maybe some type changes upon switch-in could help.

The way I set up the sets in the team generator should make it pretty easy to make changes if needed, I think.

**Comment by Joimer** [comment]
No worries, I can check balance and update this for it.

**Comment by TheImmortal** [comment]
@Joimer merge when

**Comment by ascriptmaster** [comment]
I think @Joimer still has the mindset that the 5th is when we release but we've been aiming for the 1st of the month when we can recently.

Merge whenever really but tomorrow is preferred haha. I might merge this myself before I get on the plane tomorrow, or I might not because whateverrrrrrrrrr

**Comment by Joimer** [comment]
Monday is a good day to merge and add this.

**Pull request merged**